scripts/fetch-schemas.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+curl 'https://api.entur.io/journey-planner/v2/graphql' --compressed \
```

Comment:
Burde det være noe client-name her?

Reply:
Dette kjem ikkje til å bli kjørt ofte, men eg kan sikkert legge det til for ordens skyld.